frontend/public/components/dashboard/inventory-card/inventory-item.tsx

```diff
@@ -29,11 +24,25 @@ const defaultStatusGroupIcons = {
   [InventoryStatusGroup.PROGRESS]: (
     <InProgressIcon className="co-inventory-card__status-icon--progress" />
   ),
-  [InventoryStatusGroup.NOT_MAPPED]: (
+  [InventoryStatusGroup.UNKNOWN]: (
     <QuestionCircleIcon className="co-inventory-card__status-icon--question" />
   ),
 };
 
+const getTop3Groups = (groupIDs: string[], flags: FlagsObject) => {
+  const groupStatuses = {...inventoryStatusGroupPriority};
+  plugins.registry.getDashboardsInventoryItemGroups().filter(e =>
+    Array.isArray(e.properties.required)
+      ? e.properties.required.every(flag => flags[flag])
+      : flags[e.properties.required]
```

Comment:
Use `_.castArray` to reduce complexity.

Also, we already have a helper for this in `public/components/dashboards-page/utils.ts`.

```ts
export const isDashboardExtensionInUse = (e: DashboardExtension, flags: FlagsObject) => {
  const requiredFlags = _.castArray(e.properties.required);
  return _.every(requiredFlags, (f) => flags[f]);
};
```

```ts
getDashboardsInventoryItemGroups().filter(e => isDashboardExtensionInUse(e, flags))
```

Reply:
reused `isDashboardExtensionInUse`